### ISSUE TITLE
Android Kotlin Fundamentals codelab: 

### ISSUE THREAD

**Issue by sararace** [google-developer-training/android-kotlin-fundamentals-apps#599]
**Name of the Codelab or Codelab URL**
https://developer.android.com/codelabs/kotlin-android-training-add-navigation#4

**Describe the problem**
Step 1.3 needs to be on a new line. It's showing up to the right of the picture so it's hard to follow.

**In which lesson and step of the codelab can this issue be found?**
Lesson 5 Step 1

**How to reproduce?**
Open the page in Chrome and scroll down to step 1.3

![Screen Shot 2021-03-08 at 2 23 59 PM](https://user-images.githubusercontent.com/77028847/110389839-f2bde100-8019-11eb-82e4-c0e378afede9.png)

**Versions**
1. What version of Android Studio are you using?
2. What API level are you targeting?

**Additional information**

**codelab:** android-kotlin-fundamentals

**Comment by priyanshu003** [comment]
i want to solve this issue
